ui/raidboss/data/07-dt/raid/r8s.ts

```diff
@@ -1,14 +1,1907 @@
+      condition: (data, matches) => data.me === matches.source,
+      infoText: (_data, matches, output) => {
+        if (matches.id === headMarkerData.stoneTether)
+          return output.side!({ wolf: output.wolfOfWind!() });
+        return output.side!({ wolf: output.wolfOfStone!() });
+      },
+      outputStrings: {
+        wolfOfWind: {
+          en: 'Green',
+        },
+        wolfOfStone: {
+          en: 'Yellow',
+        },
+        side: {
+          en: '${wolf} Side',
+        },
+      },
+    },
+    {
+      id: 'R8S Tactical Pack Debuffs',
+      // Durations could be 21s, 37s, or 54s
+      type: 'GainsEffect',
+      netRegex: { effectId: ['1127', '1128'], capture: true },
+      condition: (data, matches) => {
+        return data.me === matches.target && data.phase === 'adds';
+      },
+      response: (data, matches, output) => {
+        // cactbot-builtin-response
+        output.responseOutputStrings = stoneWindOutputStrings;
+
+        // 1127 = Earthborne End (Yellow Cube) Debuff
+        // 1128 = Windborne End (Green Sphere) Debuff
+        const cubeDebuffId = '1127';
+        data.stoneWindDebuff = matches.effectId === cubeDebuffId ? 'stone' : 'wind';
+
+        if (parseFloat(matches.duration) < 22) {
+          data.stoneWindCallGroup = 1;
+        } else if (parseFloat(matches.duration) < 38) {
+          data.stoneWindCallGroup = 2;
+        } else {
+          data.stoneWindCallGroup = 3;
+        }
+
+        return {
+          infoText: output.stoneWindNum!({
+            debuff: output[data.stoneWindDebuff]!(),
+            num: data.stoneWindCallGroup,
+          }),
+        };
+      },
+    },
+    {
+      // headmarkers with casts:
+      // A3CF (Pack Predation) from Wolf of Wind
+      // A3E4 (Pack Predation) from Wolf of Stone
+      // Simultaneously highest aggro gets cleaved:
+      // A3CD (Alpha Wind) from Wolf of Wind
+      // A3E2 (Alpha Wind) from Wolf of Stone
+      id: 'R8S Pack Predation',
+      type: 'HeadMarker',
+      netRegex: { id: headMarkerData.predation },
+      condition: (data) => data.phase === 'adds',
+      infoText: (data, matches, output) => {
+        data.packPredationTargets.push(matches.target);
+        if (data.packPredationTargets.length < 2)
+          return;
+
+        // Increment count for group tracking
+        data.packPredationTracker = data.packPredationTracker + 1;
+
+        const name1 = data.party.member(data.packPredationTargets[0]);
+        const name2 = data.party.member(data.packPredationTargets[1]);
+
+        return output.predationOnPlayers!({ player1: name1, player2: name2 });
+      },
+      run: (data) => {
+        if (data.packPredationTargets.length >= 2)
+          data.packPredationTargets = [];
+      },
+      outputStrings: {
+        predationOnPlayers: {
+          en: 'Predation on ${player1} and ${player2}',
+        },
+      },
+    },
+    {
```

Comment:
Should add a TODO here for a followup PR with "earth first"/"wind first" config options.

Reply:
Added.